src/Worker/Grpc/GrpcDurableTaskWorker.Processor.cs

```diff
@@ -87,26 +86,17 @@ public async Task ExecuteAsync(CancellationToken cancellation)
             }
         }
 
-        static OrchestrationRuntimeState BuildRuntimeState(P.OrchestratorRequest request)
+        static IEnumerable<T> Append<T>(IEnumerable<T> source, IEnumerable<T> items)
```

Comment:
Is this method needed anymore, or can it be replaced with `source.Concat(items)`?

Reply:
Oh, I got confused and thought the LINQ `Append` method (which is only in .NET 6+) was `Concat`. Yeah, `Concat` is what I want here. I'll remove this Append method.